### PR TITLE
Create page about using certificates

### PR DESCRIPTION
**Pull request by MichaelDeciso**
I think there must be more to write about, but I'm not sure which direction to take. I do think this is at least a good start.

### CODE REVIEW

File: source/manual/certificates.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+==================
+Using certificates
+==================
+
+In OPNsense, certificates are used for encrypting traffic. To make using them easier, OPNsense allows creating
```
Comment (fabianfrz) [inline source/manual/certificates.rst]:
```suggestion
In OPNsense, certificates are used for ensuring trust between peers. To make using them easier, OPNsense allows creating
```

File: source/manual/certificates.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+certificates from the front-end. In addition to that, it also allows creating certificates for other purposes,
+avoiding the need to use the ``openssl`` command line tool. Certificates in OPNsense can be managed from
+**System->Trust->Certificates**.
+
+The following types of certificate can be generated in OPNsense:
+
+* Client
+* Server
+* Combined Client/Server
+* Certificate Authority
```
Comment (fabianfrz) [inline source/manual/certificates.rst]: they should also be explained (especially if you configure a server with a client certificate, the connection should be rejected -> especially important for the web interface)

File: source/manual/certificates.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+In addition to this, OPNsense can generate a Certificate Signing Request (CSR). This can be used if you want to create a
+certficate signed by an external CA.
```
Comment (fabianfrz) [inline source/manual/certificates.rst]: I would mention how they are used, i. e. https://github.com/opnsense/docs/blob/master/source/manual/how-tos/nginx_tls_auth.rst for client certificates.

File: source/manual/certificates.rst (hunk 1 above)
Comment (MichaelDeciso) [reply source/manual/certificates.rst]: Hm, not sure what the best phrasing is here. @mimugmail do you have an idea?

Comment (mimugmail) [inline source/manual/certificates.rst]:
```suggestion
In OPNsense, certificate management is easy, as it allows creating
```

Comment (mimugmail) [reply source/manual/certificates.rst]: I'd make is more general as is suggestion :)

File: source/manual/certificates.rst (hunk 3 above)
Comment (fichtner) [reply source/manual/certificates.rst]: From core perspective OpenVPN and IPsec.

File: source/manual/certificates.rst (hunk 2 above)
Comment (MichaelDeciso) [reply source/manual/certificates.rst]: I have added a warning.

File: source/manual/certificates.rst (hunk 3 above)
Comment (MichaelDeciso) [reply source/manual/certificates.rst]: OK, I listed those two. Normally, I'd put 3-4 examples, are there more or should I just leave it at two?

Comment (fichtner) [reply source/manual/certificates.rst]: Captive Portal and web proxy then. While the admin GUI uses it too it should stay on topic for user-facing parts, not admin parts.

File: source/manual/certificates.rst (hunk 1 above)
Comment (fabianfrz) [reply source/manual/certificates.rst]: @MichaelDeciso at least not "encrypting traffic" as this is done by a cipher and never by a certificate.

File: source/manual/certificates.rst (hunk 2 above)
Comment (fabianfrz) [reply source/manual/certificates.rst]: @MichaelDeciso perfect. Well written :+1:

File: source/manual/certificates.rst (hunk 1 above)
Comment (MichaelDeciso) [reply source/manual/certificates.rst]: In the end, I just went with your suggestion.

**Pull request merged**